### PR TITLE
First Iteration of /GAME message translations

### PR DESCRIPTION
**Pull request by fedprod**

### CODE REVIEW

File: Fallout2/Fallout1in2/mods/fo1_german/text/german/game/quests.msg [hunk 1]
```diff
@@ -13,33 +13,33 @@
...
 {104}{}{Zerst�re die Quelle der Mutanten.}	#fo1 vanilla
-{105}{}{Help the medic to synthesize Antidote.}
-{106}{}{Learn about the outside world at the library.}
-{107}{}{Gain entry to the Armory.}
-{108}{}{Gain entry back into the Vault.}
-{109}{}{Kill the Overseer.}
-{110}{}{Kill all inhabitants of Vault 13.}
-{120}{}{Find the Water Chip.}
+{105}{}{Hilf dem Sanit�ter ein Gegengift herzustellen.}
+{106}{}{Lerne aus den Vault-Archiven �ber die Aussenwelt.}
```
Comment (Lexx2k) [inline Fallout2/Fallout1in2/mods/fo1_german/text/german/game/quests.msg]: Außenwelt :>

**Comment by wipe2238** [comment]
There are some duplicated entries in WORLDMAP.MSG, see [here](https://github.com/rotators/Fo1in2/pull/28/checks?check_run_id=426343819#step:3:1). Looks like copypaste fail in tables 54 and 61.

File: Fallout2/Fallout1in2/mods/fo1_german/text/german/game/quests.msg (hunk 1 above)
Comment (fedprod) [reply Fallout2/Fallout1in2/mods/fo1_german/text/german/game/quests.msg]: Ich blame die Rechtschreibreformen >.<

**Comment by fedprod** [comment]
> 
> 
> There are some duplicated entries in WORLDMAP.MSG, see [here](https://github.com/rotators/Fo1in2/pull/28/checks?check_run_id=426343819#step:3:1). Looks like copypaste fail in tables 54 and 61.

Actually a fail in the original vanilla german tables. Let me check the tables with the EN ones quickly again...

**Comment by wipe2238** [comment]
> Also a fail in the vanilla english ones

Yeah, they aren't pretty either, i remember we was fixing .msg from vanilla FO2 in other mod as well. And sometimes screwup in .msg can end in hard to track crashes, that's why so strict checks...

Anyway, thanks 👍

**Comment by Lexx2k** [comment]
The thing is, none of these strings are used by et Tu, so translating them was kinda pointless. :> I've already went over the file before and as far as I could tell, everything important was translated. The bigger issue are dialog files themselves.

**Comment by fedprod** [comment]
Sure, but that way nobody can say something is missing in the future xP
The few hours weren't a biggie anyhow. Dialogue files are next of course

**Comment by Lexx2k** [comment]
I have marked untranslated lines in the current dialog files with TODO: TRANSLATE so they should be easy to find. However, there's other .msg files that I haven't moved to text/german/ yet, because I don't really know what's really used and what is deadweight. It's probably better to check ingame if there are any critters or objects that still need to be translated. 

What I know for sure is that most of the invasion mutant stuff made by TeamX doesn't have a translation.

**Comment by burner1024** [comment]
I'll leave [this link](https://tra.bgforge.net/) here, just in case anyone's interested.

**Pull request merged**